docs/installation.rst

```diff
@@ -122,6 +122,22 @@ After all dependencies have been installed, we can build the code:
         cmake ..
         make
 
+Configuring the dynamics
+------------------------
+The dynamics for nextSIM are chosen at the point of configuring CMake. This is in contrast to most of the model configuration, which is done at model run time. The dynamics are set through the configuration option ``DynamicsType``. The available options for the dynamics are
+
+* ``DG1``: First order discontinuous Galerkin dynamics. Advection calculations are performed with 3 DG components.
+
+  * This is the default option is no other option is provided to CMake.
+
+* ``DG2``: Second order discontinuous Galerkin dynamics. Advection calculations are performed with 6 DG components.
```

Comment:
Again, could mention that this is for a 2D triangular mesh.

Reply:
Added '2D rectangular grid'.